deployToAzure.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+echo "Initialise terraform for Azure"
+terraform -chdir=terraform init  -backend-config "resource_group_name=$RESOURCE_GROUP_NAME" -backend-config "storage_account_name=tfstate14106" -backend-config "key=terraform.tfstate" -backend-config "container_name=tfstate" -backend-config "access_key=$ACCOUNT_KEY"
+
+echo "Apply terraform for Azure"
+terraform -chdir=terraform apply -auto-approve -var 'prefix=dev' -var 'location=uksouth' -var "client_id=$ARM_CLIENT_ID" -var 'storage_account_name=tfstate14106' -var "client_secret=$ARM_CLIENT_SECRET" -var "resource_group_name=$RESOURCE_GROUP_NAME" -var "access_key=$ACCOUNT_KEY" -var 'key=terraform.tfstate' -var 'container_name=tfstate'
```

Comment:
`client_id` and `client_secret` are supposed to be taken from your OAuth app registration, so your app can identify itself to GitHub.

ARM_CLIENT_ID and ARM_CLIENT_SECRET are for the Azure Service Principal and are automatically picked up by terraform for auth.

So currently your OAuth doesn't work. https://dev-dirtytodo2021.azurewebsites.net/

Reply:
Done